data/org.ArcticaProject.arctica-greeter.gschema.xml

```diff
@@ -211,6 +211,10 @@
       <default>false</default>
       <summary>Whether to hide Wayland sessions.</summary>
     </key>
+    <key name="preferred-sessions" type="as">
+      <default>[]</default>
```

Comment:
Shouldn't this list really be `['mate', 'xfce', 'kde-plasma', 'kde', 'gnome', 'cinnamon', 'lomiri']` to keep the old default?

Without that, and manual configuration, the new behavior will be to always fall back to the default lightdm session instead of scanning for the sessions we previously scanned for.

Reply:
The actual original default was no specific preference list.

Reply:
So, we are returning back to the original functionality.